content/_data/changelogs/weekly.yml

```diff
@@ -5925,6 +5925,8 @@
     references:
       - url: https://github.com/jenkinsci/jep/blob/master/jep/222/README.adoc
         title: JEP-222
+      - url: /blog/2020/01/27/web-socket/
```

Comment:
Since no one seems to be around today, feel free to update to the current date as well as renaming `2020-01-27-web-socket.adoc` when actually publishing.

Reply:
I'm running my Chromebook from a hotel room currently and don't know the GitHub web UI technique to rename a file in a pull request.  I propose that we accept this blog post date "as is" and publish today, 29 Jan 2020.  Close enough to the actual date of publication for my concerns.

Reply:
Will update the date for today.